### PR TITLE
Support assigning a String to a style property

### PR DESCRIPTION
**Pull request by Atry**

### CODE REVIEW

File: src/main/scala/org/scalajs/dom/raw/Html.scala [hunk 1]
```diff
@@ -3968,7 +3969,8 @@ abstract class HTMLElement extends Element {
    *
    * MDN
    */
-  var style: CSSStyleDeclaration = js.native
+  def style: CSSStyleDeclaration = js.native
+  def style_=(value: String | CSSStyleDeclaration): Unit = js.native
```
Comment (sjrd) [inline src/main/scala/org/scalajs/dom/raw/Html.scala]: I think overloads would be more appropriate, here.

**Comment by sjrd** [comment]
OK, could you squash the three commits, and keep only the first commit's message?

**Comment by Atry** [comment]
@sjrd Done. Waiting for CI now.

**Comment by sjrd** [comment]
LGTM

**Comment by Atry** [comment]
@sjrd Some of Travis CI jobs failed due to network problem: https://travis-ci.org/scala-js/scala-js-dom/jobs/163351090

**Pull request merged**